src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/IverksettMotOppdragTask.kt

```diff
@@ -35,7 +39,7 @@ class IverksettMotOppdragTask(val hentIverksettService: HentIverksettService, va
             forrigeTilkjentYtelse
         )
 
-        // TODO : Lagre denne (utbetaling, som en json)
+        lagreTilstandService.lagreTilkjentYtelseForUtbetaling(behandlingId = behandlingId, utbetaling)
```

Comment:
Hva er det tenkte flytet med tilkjent ytelse her? 
Er tanken att `forrigeTilkjentYtelse` skal plukkes opp fra `tilstands`-tabellen?

Reply:
Ja, forstår det som at den skal hentes opp ? Det er en TODO på det i alle fall. Skal man da sjekke om det finnes en fra før med samme behandlingsId, og så hente denne ? Hva skjer hvis man skal hente opp en gang til ? Må man slette dem etterhvert, eller sortere dem ? Eller skjer det ikke at man henter forrige tilkjent ytelse mere enn én gang ?

Reply:
Det var min opprinnelige tanke iallefall, men er lydhør hvis det finnes smartere måter å håndtere det på. Men vi trenger jo "forrige" på et eller annet tidspunkt for å sette riktige perioder. Og et eller annet sted må vel det lagres ?

Reply:
Ja jeg tror nok det blir mest riktig, ville bare høre :D